File: pingyougou_cart_interface/src/main/java/com/pingyougou/cart/service/CartService.java
package com.pingyougou.cart.service;



import com.pingyougou.groupEntity.Cart;

import java.util.List;

public interface CartService {

    /**
     * 添加商品到购物车列表
     */
    public List<Cart> addItemToCartList(List<Cart> cartList, Long itemId, Integer num);

    List<Cart> selectCartListFromRedis(String sessionId);

    void saveCartListToRedis(String sessionId, List<Cart> cartList);

    List<Cart> mergeCartList(List<Cart> cartList_sessionId, List<Cart> cartList_username);

    void deleteCartList(String sessionId);
}

File: pingyougou_seckill_web/src/main/webapp/js/service/cartService.js
//服务层
app.service('cartService',function($http){
	    	
	//读取购物车列表数据
	this.findCartList=function(){
		return $http.get('cart/findCartList.do');
	}

	//添加商品到购物车
	this.addItemToCartList=function (itemId,num) {
        return $http.get('cart/addItemToCartList.do?itemId='+itemId+"&num="+num);
    }
});
